CSS/typography.css

```diff
@@ -69,11 +69,17 @@
 .border-rad-full{
     border-radius: 62.4375rem;
 }
-
-/* .test-border{
-    width: 10rem;
-    height: 10rem;
+/* text - color */
```

Comment:
such comments are not really needed as the name of the class is descriptive enough